PROJECT-FLOW.md

```diff
@@ -51,18 +53,19 @@
 - Group the poll is a part of **ForeignKey, group_id** _( many-to-one relationship)_
 - Options **Database relationship to options schema** _(One to many Relationship)_
 - array of votes _( many-many relationships )_
-- result ( ? ) **string(300)** _(Ambiguity: If we store it in plain someone not part of the group will be able to see the result. If we encrypt it with the admin's master key, the admin might be able to manipulate the result)_
+- result ( ? ) **string** _Plain Text_
 
   #### 3. a. Association Object For Votes
 
   - left_id **user_id**
   - right_id **poll_id**
-  - Encrypted Vote **String(300)**
+  - Encrypted Vote **varbinary**
 
 ### 4 Options Schema
 
-- option*id **poll_id ( primary key )** *( Poll to which the option belongs to )*
-- option-description **string(50)**
+- option_id **primary key** *( Option-code for aiding in MPC)*
+- poll_id **poll_id ( primary key )** *( Poll to which the option belongs to )*
+- option-description **string**
```

Comment:
created a unique key (option_id) for the options schema to identify the options.